sdk/servicebus/Microsoft.Azure.ServiceBus/src/Primitives/ManagedIdentityTokenProvider.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.Azure.ServiceBus.Primitives
     /// <summary>
     /// Represents the Azure Active Directory token provider for Azure Managed Service Identity integration.
```

Comment:
If you're moving forward with the rename, you'll probably want to consider updating this comment as well.